pkg/sink/codec/common/config.go

```diff
@@ -76,6 +76,8 @@ type Config struct {
 
 	// for open protocol
 	OnlyOutputUpdatedColumns bool
+	// Whether old value should be excluded in the output.
+	OpenOutputOldValue bool
```

Comment:
This should only control the `open-protocol`?

Reply:
yes